step-definitions/core/core.js

```diff
@@ -6,46 +6,54 @@ import { setupEvan } from '../../test-utils/test-utils.js';
 export const WAIT_TIME = 15 * 1000;
 
 When(/^I click the element with id "([^"]+)"$/, async id => {
+  client.useCss();
```

Comment:
client.useCss(); should be the default behaviour and not necessary to be set. If so there is probably some test using xPath and not resetting it to css afterwards.